Makefile

```diff
@@ -734,7 +734,11 @@ ocamlopt.opt: compilerlibs/ocamlcommon.cmxa compilerlibs/ocamloptcomp.cmxa \
               $(OPTSTART:.cmo=.cmx)
 	$(CAMLOPT_CMD) $(LINKFLAGS) -o $@ $^
 
-ocamlopt-memtrace.opt: otherlibrariesopt
+ocamlopt-memtrace.opt: compilerlibs/ocamlcommon.cmxa \
+                       compilerlibs/ocamloptcomp.cmxa \
+                       driver/optmaindriver.cmx \
+                       otherlibrariesopt libraryopt \
+                       memprof/memtrace
```

Comment:
It seems unnecessary to have all those `cmx`/`cmxa` files here. I would put `ocamlopt.opt`, to specify that they should be kept in sync, and `otherlibrariesopt`, as it is an additional dependency of the memtrace version, and that's all.

Reply:
done